.github/workflows/ci.yml

```diff
@@ -54,7 +54,9 @@ jobs:
     steps:
     - name: Setup MySQL latest
       if: matrix.db-type == 'mysql' && matrix.php-version == '7.4'
-      run: docker run --rm --name=mysqld -e MYSQL_ROOT_PASSWORD=root -e MYSQL_DATABASE=cakephp -p 3306:3306 -d mysql --default-authentication-plugin=mysql_native_password --disable-log-bin
+      run: |
+        sudo service mysql start
+        mysql -h 127.0.0.1 -u root -proot -e 'CREATE DATABASE cakephp;'
```

Comment:
This is also backported from 5.x due to errors in 'prefer-lowest' CI inspection.